acceptance/config/gem/options.rb

```diff
@@ -1,9 +1,7 @@
 {
   :type => 'git',
```

Comment:
should this be `:type => 'gem'`?

Reply:
This is intentionally set to `git` so that `git` and `gem` have the same ruby environment. I'll add a note.